### DISCUSSION TITLE
Use custom model as product

### DISCUSSION THREAD

**Discussion by xujiongze** [lunarphp/lunar#1675] [Q&A] [answered]
Hello all,
I already have a project, which is about auctions.
And I want to use the auction lot directly as the product instead of using lunar's product.

Is possible to "implement purchasable" to my LOT model and work like product ? I looked in the documentation but there is nothing related

**Comment by glennjacobs** [comment, marked as answer]
In theory, yes. But I don't think many have attempted it, so I'm unsure as to what issues you may come up against. So probably best try a simpler test project to see what you think.

**Reply by xujiongze** [reply]
Thanks for quickly reply, I will try

**Comment by costafrancesco94** [comment]
I'm planning to create a new `ProductType` model to achieve a custom "event booking". Is this something that is on the roadmap with other names? Would you be interested in merging and have it in the core?
